templates/projects/splash.macros.j2

```diff
@@ -25,27 +33,17 @@
   {% set client = page.config.client[0] if page.config.client else none %}
   {% set contrib = page.config.contributors[0] if page.config.contributors else none %}
 
-  {% set slug = ['/', page.slug[:-6] if page.slug.endswith('/index') else page.slug, '/']|join %}
-  {% set link = slug if (project.owner or project.case_study) else '/work/' %}
-  {% set link = '/open-source/' if (contrib and link == '/work/') else link %}
-
-  {% set images = page.config.image or none %}
+  {% set url = ('/' + page.slug|replace('/index', '') + '/') %}
+  {% set type = '/open-source/' if contrib else '/work/' %}
+  {% set link = url if (project.owner or project.case_study) else type %}
 
-  {% if images %}
-    {% set splash = images|selectattr('type', 'equalto', 'splash')|list or none %}
-    {% set image = splash[0].src if splash else images[0].src %}
-    {% set image = image if '://' in image else ['/static/images/', image]|join('') %}
-
-    <a href="{{ link }}" class="splash-project">
-      <div class="splash-image" style="background-image: url('{{ image }}');"></div>
-
-      <h2 class="splash-client">
-        {{ client.name if client else project.name }}
-      </h2>
-
-      <h3 class="splash-title">
-        {{ project.name if client else [birds.get_bird_name(contrib.author), contrib.role]|join(', ') }}
-      </h3>
+  <article class="splash-project">
+    <a href="{{ link }}" class="splash-logo">
+      {% include 'projects/_%s.svg' % (page.config.logo or 'mozilla') %}
```

Comment:
Why the specific fallback to `'mozilla'`?

Reply:
oh, that was just for my testing before I had other things in place :p